include/openPMD/RecordComponent.hpp

```diff
@@ -430,4 +525,103 @@ RecordComponent::storeChunk(T_ContiguousContainer & data, Offset o, Extent e)
 
     storeChunk(shareRaw(data), offset, extent);
 }
+
+template< typename T, typename F >
+inline Span< T >
+RecordComponent::storeChunk( Offset o, Extent e, F && createBuffer )
```

Comment:
Is it worth adding a doxygen string to this overload that shows a usage example or so?